dust/directive.py

```diff
@@ -29,7 +29,7 @@ def depart_review_node(self, node):
     self.depart_admonition(node)
 
 
-class ReviewerMetaDirective(rst.Directive):
+class ReviewerMetaDirective(Note):
```

Comment:
I think we should stick with `rst.Directive`. From experimentation I've done, I don't see any visual difference when inheriting from `Note`, but it allows the `reviewer-meta` directive to have a body:

```rst
.. reviewer-meta::
    :written-on: 2016-06-21
    :proofread-on: 2016-06-21
    
    Foo bar baz
```           

This is an error when simply inheriting from `rst.Directive`. Since we don't use the content of the directive, allowing it to build without errors gives the assumption that it'll be used, which is wrong.